feedfetcher.py

```diff
@@ -68,8 +69,9 @@ def post_text(text, username, channel, iconurl):
                 else:
                     if not silent_mode:
                         logging.debug('Nothing new. Waiting for good news...')
-            except:
-                logging.critical('Error fetching feed.')
-                continue
+                except Exception as e:
```

Comment:
sorry, it is corrected with another commit 056e433 below
Do I need to update the pull request anyhow?